src/code/reducers/index.js

```diff
@@ -134,6 +134,14 @@ export default function reducer(state, action) {
         correct: state.correct + 1
       });
     }
+    case actionTypes.ZOOM_CHALLENGE_WON: {
+      let progress = updateProgress(state, true);
```

Comment:
I don't think the second parameter is necessary here, but probably not worth changing since this is all getting refactored soon anyway